canonical-web-and-design/canonicalwebteam.image-template
# Update instructions to support native lazy

I'm making use of this new version (`v1.0.0`) in https://github.com/canonical-web-and-design/ubuntu.com/pull/5895 - review that PR to QA these changes.

### Changes

Support for the `loading` attribute to use native lazyloading support in Chrome and other browsers when they have it.

Remove the need for extra noscript CSS by using the [lazysizes noscript plugin][1], which also allows us to simplify the markup.

Change the interface to take a `loading` rather than `lazy` attribute, to stay closer to the HTML standard.

Change the interface to remove `extra_classes` and add an explicit `attrs` dictionary, rather than `**attributes`. This makes it clearer for the user what's happening, as they can now simply pass in `attrs={"class": "my-class"}`, which will be directly output in the markup.

Change the README to Markdown format (much nicer for everybody).

QA
--

see https://github.com/canonical-web-and-design/ubuntu.com/pull/5895


[1]: https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/noscript
